packages/files-ui/cypress/tests/search-spec.ts

```diff
@@ -0,0 +1,67 @@
+import { homePage } from "../support/page-objects/homePage"
+import { folderName, folderPath } from "../fixtures/filesTestData"
+import { searchPage } from "../support/page-objects/searchPage"
+import { apiTestHelper } from "../support/utils/apiTestHelper"
+
+describe("Search", () => {
+
+  context("desktop", () => {
+
+    it("can search for files and folders via name", () => {
+      cy.web3Login({ clearCSFBucket: true, clearTrashBucket: true })
+
+      // add file and folders
+      homePage.uploadFile("../fixtures/uploadedFiles/logo.png")
+      homePage.fileItemName().invoke("text").as("fileName")
+      apiTestHelper.createFolder(`/${folderName}A`)
+      apiTestHelper.createFolder(`/${folderName}B`)
+
+      // search for a specific folder, ensure 1 result is found
+      homePage.searchInput().type(`${folderName}A{enter}`)
+      searchPage.fileItemRow().should("be.visible")
+        .should("have.length", 1)
+      searchPage.fileItemName().should("contain.text", `${folderName}A`)
+      cy.go("back")
+
+      // perform a loose search for folders, ensure that 2 results are found
+      homePage.searchInput().type(`{selectall}{del}${folderName}{enter}`)
+      searchPage.fileItemRow().should("be.visible")
+        .should("have.length", 2)
+      searchPage.fileItemName().should("contain.text", `${folderName}A`)
+      searchPage.fileItemName().should("contain.text", `${folderName}B`)
+      cy.go("back")
+
+      // search for a specific file, ensure only 1 result is found
+      cy.get("@fileName").then(($fileName) => {
+        homePage.searchInput().type(`{selectall}{del}${$fileName}{enter}`)
+        searchPage.fileItemRow().should("be.visible")
+          .should("have.length", 1)
+        searchPage.fileItemName().should("contain.text", `${$fileName}`)
```

Comment:
Not 100% sure about that one :)
```suggestion
        searchPage.fileItemName().should("contain.text", $fileName)
```